src/status_im/contexts/keycard/migrate/re_encrypting/view.cljs

```diff
@@ -0,0 +1,26 @@
+(ns status-im.contexts.keycard.migrate.re-encrypting.view
+  (:require [quo.core :as quo]
+            [react-native.core :as rn]
+            [status-im.common.resources :as resources]
+            [utils.i18n :as i18n]
+            [utils.re-frame :as rf]))
+
+(defn view
+  []
+  (let [profile-name        (rf/sub [:profile/name])
+        profile-picture     (rf/sub [:profile/image])
+        customization-color (rf/sub [:profile/customization-color])]
+    [:<>
+     [quo/page-top
+      {:title           (i18n/label :t/re-encrypting-data)
+       :description     :context-tag
+       :context-tag     {:full-name           profile-name
+                         :profile-picture     profile-picture
+                         :customization-color customization-color}
+       :container-style {:margin-top 56}}]
+     [quo/text {:style {:padding-horizontal 20}}
+      (i18n/label :t/do-not-quit)]
+     [rn/view {:style {:flex 1 :align-items :center :justify-content :center}}
```

Comment:
i don't feel it worth it honestly, i don't believe in styles reusage in such cases

Reply:
In this case it's what's sometimes called a _true duplication_ @flexsurfer, that is, it's guaranteed that a developer will need to update 3 different places whenever touching one, which is not great.

A _false duplication_ is a common source of confusion, when devs try to remove duplication just because the code is equal. This is not what I mean, I also agree with you sharing styles just for the sake of removing duplication is not the best.

Reply:
> that is, it's guaranteed that a developer will need to update 3 different places whenever touching one,

I wouldn't be so sure

i prefer to change in 3 places and make sure than change in one and be not sure where it changed , these styles so simple so no need to worry about them

Reply:
This creates some tech debt, but it's a minor thing. I believe I'm not the only dev who would touch these components and refactor a little bit @flexsurfer.

But no big deal 👍🏼